examples/event-api/server.py

```diff
@@ -6,16 +6,17 @@
 
 async def main():
     kd.init_session()
-    
+
     start = time.time()
     requestmap = dict()
 
-    # Initialize event source with historical data
-    events = kd.sources.PyList(
+    # Initialize event source with schema from historical data.
+    events = kd.sources.PyDict(
         rows = [{"ts": start, "user": "user_1", "request_id": "12345678-1234-5678-1234-567812345678"}],
         time_column = "ts",
         key_column = "user",
-        time_unit = "s"
+        time_unit = "s",
+        retained=False,
```

Comment:
disable retention in the server example, since we only care about the materialization. note that the `rows` above will then be used only for schema inference.